app/storybook/documentation/ui-kit/api/AverageMetricQueryInput.mdx

```diff
@@ -0,0 +1,8 @@
+import { ArgTypes, Meta } from '@storybook/blocks'
```

Comment:
The page in Storybook is incorrect
<img width="982" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/4bf736b8-b2d7-4762-9819-7259026db3a6">

Reply:
False alarm! I've re-installed and re-built the project and it looks good now. Sorry!

<img width="985" alt="image" src="https://github.com/propeldata/ui-kit/assets/55801864/9429dcbf-5566-40e3-89d7-c4ea1b95cc1f">